tests/test_streaming.py

```diff
@@ -214,15 +214,6 @@ async def ret_a(x: str) -> str:
                 ModelRequest(
                     parts=[ToolReturnPart(tool_name='ret_a', content='hello world', timestamp=IsNow(tz=timezone.utc))]
                 ),
-                ModelRequest(
-                    parts=[
-                        ToolReturnPart(
-                            tool_name='final_result',
-                            content='Final result processed.',
-                            timestamp=IsNow(tz=timezone.utc),
-                        )
-                    ]
-                ),
```

Comment:
Why is this dropped?

Reply:
because we can't add this part/message until the stream is complete, so it get's added later.